samples/drivers/spi_bitbang/README.rst

```diff
@@ -0,0 +1,54 @@
+.. _spi-bitbang-sample:
+
+SPI-Bitbang Sample
+####################
+
+Overview
+********
+
+This sample demonstrates using the bitbang SPI driver. The bitbang driver can
+be useful for devices which use a non multiple of 8 word size, for example some
+LCDs which have an extra cmd/data bit.
+
+This sample loops through some different spi transfer configurations.
+
+
+Building and Running
+********************
+
+The application will build only for a target that has a :ref:`devicetree
+<dt-guide>` entry with ``zephyr,spi-bitbang`` as a compatible.
```

Comment:
```suggestion
<dt-guide>` entry with :dtcompatible:`zephyr,spi-bitbang` as a compatible.
```

This is preferable because it creates a link to the binding's autogenerated documentation in the [bindings index](https://docs.zephyrproject.org/latest/reference/devicetree/bindings.html).